functions/analyze/analyze.test.js

```diff
@@ -0,0 +1,26 @@
+import analyze from './analyze';
+
+test('Gets average', () => {
+  expect(analyze([1, 8, 3, 4, 2, 6]).average).toEqual(4);
+});
+
+test('Gets minimum value', () => {
+  expect(analyze([1, 8, 3, 4, 2, 6]).min).toEqual(1);
+});
+
```

Comment:
### [optional changes]
Try to add some tests that test negative scenario
exampe: analyze([1, 8, 3, 4, 2, 6]) to not equal 16